src/docbuilder/metadata/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "metadata"
```

Comment:
If this is going to be published, it needs a less generic name.

Reply:
Done, I changed it to `docsrs-metadata`.